scripts/build-all-bottles.sh

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env nix-shell
+#! nix-shell shell.nix -i bash
+
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+set -euo pipefail
+
+if [ -z "$1" ] ; then
+    echo "Please call this script with the name of the OS for which to build the bottles."
+    exit 1
+fi
+
+# we don't bottle meta-formulas that contain only services
+formulae=("tezos-accuser-011-PtHangz2" "tezos-accuser-012-Psithaca" "tezos-admin-client" "tezos-baker-011-PtHangz2" "tezos-baker-012-Psithaca" "tezos-client" "tezos-codec" "tezos-endorser-011-PtHangz2" "tezos-node" "tezos-sandbox" "tezos-signer")
+
+# we filter out only the formulas that were changed and so need to be rebuilt
+changed_formulae=()
```

Comment:
Nitpick: I think that "changed" is slightly confusing. This script needs to be restarted when e.g. there are issues with the network connection to the builder. So I'd rather say that we filter out formulae that don't have bottles in the corresponding release